docs/content/how-to/agent/create-agent-cluster.md

```diff
@@ -554,7 +554,7 @@ EOF
 
 #### Step 2 - Get the MetalLB Operator Configured
 
-We will create an `IPAddressPool` with a single IP address and L2Advertisement to advertise the LoadBalancer IPs provided by the `IPAddressPool` via L2. 
+We will create an `IPAddressPool` with a single IP address and BGPAdvertisement to advertise the LoadBalancer IPs provided by the `IPAddressPool` via BGP protocol.
```

Comment:
Nope, TMK kubevirt is using cloud provider KubeVirt to provide Load Balancer for the hosted cluster

Reply:
yes, but cloud provider kubevirt is using metalLB on the hosting cluster to do that.

Reply:
Sorry, I'm not familiar with how kubevirt provides the LoadBalancer service.
This change to the Agent docs is required because the worker nodes can be on a different L2 from the management cluster. 
We noticed that in some cases the L2Advertisement isn't working for the ingressOperator (running on the management cluster) and it's failing to perform the Canary checks.
That's why we replaced  L2Advertisement with BGPAdvertisement.

Reply:
ah, I read the documentation closer, and i think i misunderstood where metal lb was being configured.

I believe you're setting up metalLB here within the hosted cluster, which is not something kubevirt does.  We do however use metalLB on the underlying hosting (mgmt/infra) cluster that hosts the kubevirt VMs though. I don't think your changes here impact us though

sorry for the confusion... well. this stuff is confusing :D